client/src/components/FoodSeeker/AddressDropDown.jsx

```diff
@@ -121,16 +122,21 @@ export default function AddressDropDown({ autoFocus }) {
   };
 
   const handleKeyDown = (event) => {
-    if (event.key === "Enter" && mapboxResults.length > 0) {
+    if (event.key === "Enter" && mapboxResults.length > 0 && !isLoading) {
       event.preventDefault();
-      handleAutocompleteOnChange(mapboxResults[0].place_name);
+      const selected = highlightedOption ?? mapboxResults[0].place_name;
+      handleAutocompleteOnChange(selected);
     }
   };
   return (
     <>
       <Autocomplete
+        disableCloseOnSelect
```

Comment:
Hi @VirginiaWu11, my expectation was the dropdown wouldn't close if I select an option but it closes automatically. Please share the default behavior for this. Thank you.